Makefile

```diff
@@ -223,9 +223,11 @@ fmt:
 	@goimports -l $(CHECKDIRS) && goimports -w $(CHECKDIRS)
 	@gofmt -s -l -w $(CHECKSRCS)
 
+# TODO: Remove the cli folder exclusion when we're done
 .PHONY: lint
 lint:
 	@gometalinter --deadline=10m --concurrency=1 --enable-gc --vendor --exclude=vendor --exclude=\.pb\.go \
+	    --exclude=cli \
```

Comment:
I'm glad that you actually caught the omission in the search criteria, but I don't feel comfortable with ignoring lint checks; we should be adding code as we go that meets our quality criteria ... I know we're trying to get all the account code merged in quickly, but how long do you anticipate a need to ignore lint issues?